app/controllers/users_controller.rb

```diff
@@ -0,0 +1,46 @@
+class UsersController < ApplicationController
+  def show
+    find_user
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to user_path(@user)
+    else
+      respond_with(@user)
+    end
+  end
+
+  def edit
+    find_user
+  end
+
+  def update
+    find_user
+    if @user.update(user_params)
+      redirect_to user_path(@user)
+    else
+      respond_with(@user)
+    end
+  end
+
+  def destroy
+    find_user.delete
+    redirect_to root_path
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:name, :email, :password, :uid)
+  end
+
+  def find_user
```

Comment:
I rally just call this method the name of the object I am looking for: i.e. `def user`

Reply:
Aah, I knew we'd talked about this method but I couldn't remember how you said to name it, thanks Elle!